src/main/java/com/company/Main.java

```diff
@@ -0,0 +1,44 @@
+package com.company;
+
+import com.company.model.Satellite;
+import com.company.service.FileReaderService;
+import com.company.service.FilterService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.stereotype.Component;
+import java.util.List;
+
+@SpringBootApplication
+public class Main implements CommandLineRunner {
+    private final FileReaderService fileReaderService;
+    private final FilterService filterService;
+    @Autowired
+    public Main(FileReaderService fileReaderService, FilterService filterService) {
+        this.fileReaderService = fileReaderService;
+        this.filterService = filterService;
+    }
```

Comment:
**Constructor Injection:** Using constructor injection for `FileReaderService` and `FilterService` excellent practice for dependency injection.👍🏻